eng/common/scripts/Verify-Links.ps1

```diff
@@ -167,6 +171,13 @@ function CheckLink ([System.Uri]$linkUri)
   }
   $checkedLinks[$linkUri] = $true;
 }
+$ReplacementPattern = "`${1}$branchReplacementName`$3"
```

Comment:
Lets only have `$1` and `$2` to make things a little easier for other folks that might use a regex. It is a little harder to remember if we have a `$3` as well.